angelleye-includes/express-checkout/class-wc-gateway-paypal-express-helper-angelleye.php

```diff
@@ -534,7 +534,7 @@ public function ec_add_body_class($classes) {
             if (!class_exists('WooCommerce') || WC()->session == null) {
                 return $classes;
             }
-            $paypal_express_terms = WC()->session->get('paypal_express_terms');
+            $paypal_express_terms = angelleye_get_session('paypal_express_terms');
             if ($this->ec_is_checkout() && $this->function_helper->ec_is_express_checkout()) {
                 $classes[] = 'express-checkout';
                 if ($this->show_on_checkout && isset($paypal_express_terms)) {
```

Comment:
@kcppdevelopers Here you are using the condition isset($paypal_express_terms), that will always evaluate to true even if the function return false. So I think we need to change it.